rbac/management/audit_log/model.py

```diff
@@ -138,3 +153,17 @@ def log_edit(self, request, resource, object):
         self.action = AuditLog.EDIT
         self.tenant_id = self.get_tenant_id(request)
         super(AuditLog, self).save()
+
+    def log_group_assignment(self, request, resource, object, type_dict, user_type):
```

Comment:
resource and object names are confusing 

it looks that resource is rather `resource_type` and then object can be `resource` :) 

Does it make sense ?